R/robustbayesianmetaanalysis.R

```diff
@@ -616,22 +613,29 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
     if (!is.null(dataset)) {
       return(dataset)
     } else{
-      var_names <-
+      varNames <-
         c(
-          options[["input_t"]],
-          options[["input_ES"]],
-          options[["input_SE"]],
-          options[["input_N"]],
-          options[["input_N1"]],
-          options[["input_N2"]],
-          unlist(options[["input_CI"]])
+          options[["inputT"]],
+          options[["inputES"]],
+          options[["inputSE"]],
+          options[["inputN"]],
+          options[["inputN1"]],
+          options[["inputN2"]],
+          unlist(options[["inputCI"]])
         )
-      var_names <- var_names[var_names != ""]
+      varNames <- varNames[varNames != ""]
       
       dataset <- readDataSetToEnd(
-        columns.as.numeric = var_names,
-        columns = if (options[["input_labels"]] != "") options[["input_labels"]]
+        columns.as.numeric = varNames,
+        columns = if (options[["inputLabels"]] != "") options[["inputLabels"]]
       )
+
+      if (options[["inputLabels"]] != ""){
+        dataset[[.v(options[["inputLabels"]])]] <- as.character(dataset[[.v(options[["inputLabels"]])]])
+        if (!validUTF8(dataset[[.v(options[["inputLabels"]])]]))
```

Comment:
Is this checked before the analysis is run now?

Reply:
yes, it's checked on data load